moonsense/client.py

```diff
@@ -23,13 +23,19 @@
 import os
 import requests
 
+from enum import Enum
 from google.protobuf import json_format
 from typing import Iterable, List
 
 from .models import Session, Chunk, TokenSelfResponse, \
     DataRegionsListResponse, SessionListResponse, ChunksListResponse, \
     CardListResponse, Card, SealedBundle
 
+class Platform(Enum):
```

Comment:
The platform should already be available here - 

https://github.com/moonsense/python-sdk/blob/main/moonsense/models/common_v2_pb2.py#L61

Should we re-use it?

Reply:
We discussed this in private and we opted for exposing the Enum here again in the native Python way. The generated way has some limitations that make it weird to use.